LibreNMS/Alert/Transport/Telegram.php

```diff
@@ -79,11 +84,18 @@ public static function configTemplate()
                     'name' => 'telegram-token',
                     'descr' => 'Telegram Token',
                     'type' => 'text',
+                ],
+                [
+                    'title' => 'Format',
+                    'name' => 'telegram-format',
+                    'descr' => 'Telegram format',
+                    'type' => 'text',
                 ]
             ],
             'validation' => [
                 'telegram-chat-id' => 'required|string',
                 'telegram-token' => 'required|string',
+                'telegram-format' => 'required|string',
```

Comment:
You shouldn't make this required as it's an optional field to telegram.

Reply:
Ok, thanks a lot @laf. Living and learning, this is (or at least, trying to be :)   ) my first contribution to this project.